src/js/_store/modules/categories.js

```diff
@@ -8,13 +8,56 @@ export default {
       name: '',
     },
     categoryList: [],
+    isConecting: false,
+    doneMessage: '',
+    errorMessage: '',
   },
   mutations: {
     doneGetAllCategories(state, payload) {
       state.categoryList = [...payload.categories];
     },
+    updateInputCategories(state, event) {
+      state.category.name = event;
+    },
+    toggleIsConecting(state) {
+      state.isConecting = !state.isConecting;
+    },
+    deleteCategoryName(state) {
+      state.category.name = null;
+    },
+    displayDoneMessage(state) {
+      state.doneMessage = 'カテゴリー作成しました！';
+    },
+    clearMessage(state) {
+      state.doneMessage = null;
+      state.errorMessage = null;
+    },
+    displayErrorMessage(state, errorObject) {
+      state.errorMessage = errorObject.message;
+    },
```

Comment:
[imo]
分割代入でも受け取れます！
```suggestion
    displayErrorMessage(state, { message }) {
      state.errorMessage = message;
    },
```

Reply:
分割代入で受け取りました！
https://github.com/gizumo-inc/gizumo_wiki_lesson/pull/27/commits/060e45ce9f43a474344c8c57708de6b18565f478